app/settings/streamingpreferences.h

```diff
@@ -63,6 +63,7 @@ class StreamingPreferences : public QObject
     enum Language
     {
         LANG_AUTO,
+        LANG_DE,
```

Comment:
This needs to go at the end of the list, otherwise it will cause existing language preferences to change due to renumbering of existing enum values.

Reply:
On second thought, I'll just fix this after merging and add a comment here so others don't make the same mistake.